src/routes/user/userPageLoadingPlaceholders.svelte

```diff
@@ -0,0 +1,9 @@
+<div class="relative pt-1">
+    <div class="mb-4 placeholder animate-pulse w-60 h-8" />
+    <div class="space-y-2">
+        {#each { length: 3 } as _}
+            <div class="placeholder animate-pulse h-10"></div>
+        {/each}
+    </div>
+    <div class="placeholder animate-pulse w-60 h-12 absolute right-0 mt-2" />
```

Comment:
Habe jetzt schon [US 167](https://tree.taiga.io/project/julius-boettger-voycar/us/167) erstellt, aber würde schonmal stark beeinflussende Klassen vorne hinschreiben.
```suggestion
    <div class="placeholder animate-pulse w-60 h-8 mb-4" />
    <div class="space-y-2">
        {#each { length: 3 } as _}
            <div class="placeholder animate-pulse h-10"></div>
        {/each}
    </div>
    <div class="placeholder animate-pulse absolute w-60 h-12 right-0 mt-2" />
```